src/pipelines/mod.rs

```diff
@@ -225,11 +241,40 @@ impl AssetFile {
     ///
     /// The base file name (stripped path, without any parent folders) is returned if the operation
     /// was successful.
-    pub async fn copy(&self, to_dir: &Path, with_hash: bool) -> Result<String> {
-        let bytes = fs::read(&self.path)
+    pub async fn copy(
+        &self,
+        to_dir: &Path,
+        with_hash: bool,
+        minify: bool,
+        file_type: AssetFileType,
+    ) -> Result<String> {
+        let mut bytes = fs::read(&self.path)
             .await
             .with_context(|| format!("error reading file for copying {:?}", &self.path))?;
 
+        bytes = if minify {
+            match file_type {
+                AssetFileType::Css => minify_html::minify(&bytes, &Cfg::spec_compliant()),
```

Comment:
Here, `Cfg::spec_compliant()` returns a `minify_css` default of `false`. Was this intended?

Reply:
Good catch, it was not intentional